### ISSUE TITLE
Link not fully created when containing accent characters

### ISSUE THREAD

**Issue by asiminiceanu** [gregjacobs/Autolinker.js#122]
I have the following link:

```
http://blog.kodiakptt.com/blog/2015/09/18/telefónica-launches-push-to-talk-communication-technology-for-enterprise/
```
Results in:

```html
<a href="http://blog.kodiakptt.com/blog/2015/09/18/telef" target="_blank">http://blog.kodiakptt.com/blog/2015/09/18/telef</a>ónica-launches-push-to-talk-communication-technology-for-enterprise/
```
My call to the link method is:

```javascript
text = Autolinker.link(
                        message.text, 
                        {
                            twitter: false,
                            replaceFn : function(autolinker, match) {
                                var tag = autolinker.getTagBuilder().build(match);  
                                tag.setInnerHtml(match.getMatchedText());
                                return tag; 
                            }
                        }
                   );
```

**Comment by newpen** [comment]
Me too! I encounter the same situation when the link has Chinese Characters.

**Comment by urish** [comment]
Same here

**Comment by simison** [comment]
Duplicate #57

**Comment by gregjacobs** [comment]
Fixed in 0.24.0
